<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpenController">
     <property name="title">
      <string>Open Controller</string>
     </property>
     <addaction name="actionDummyController"/>
    </widget>
    <widget class="QMenu" name="menuOpenCamera">
     <property name="title">
      <string>Open Camera</string>
     </property>
     <addaction name="actionFakeCamera"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuOpenCamera"/>
    <addaction name="menuOpenController"/>
<<<<<<< HEAD
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionQuit"/>
=======
>>>>>>> master
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionOpenVideoFile"/>
    <addaction name="actionRecord"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVideo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDisplay">
   <property name="text">
    <string>Display</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/icons/media-record.svg</normaloff>:/images/icons/media-record.svg</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/icons/media-playback-stop.svg</normaloff>:/images/icons/media-playback-stop.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenVideoFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/icons/media-eject.svg</normaloff>:/images/icons/media-eject.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFakeVideo">
   <property name="text">
    <string>Fake Video</string>
   </property>
  </action>
  <action name="actionTimedCapture">
   <property name="text">
    <string>TimedCapture</string>
   </property>
  </action>
  <action name="actionNo_Cameras_Found">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No Cameras Found</string>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
  </action>
  <action name="actionDummyController">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dummy Controller</string>
   </property>
  </action>
  <action name="actionSerialController">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Serial Controller</string>
   </property>
  </action>
  <action name="actionFakeCamera">
   <property name="text">
    <string>Fake Camera</string>
   </property>
  </action>
  <action name="actionMazeController">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Arduino Maze Controller</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/icons/emblem-system.svg</normaloff>:/images/icons/emblem-system.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
